client/src/api/managment-service.ts

```diff
@@ -508,6 +563,16 @@ class ManagmentServiceApi {
     return response.data;
   }
 
+  public async getNewestReport(): Promise<
+    ReportDetailedWithIncidents
+  > {
+    await this.refreshTokenIfExpired();
+    const response = await this.axiosInstance.get(
+      '/api/v1/reports/newest',
+    );
+    return response.data;
+  }
+
```

Comment:
Consider calling it latests report instead

Reply:
good idea